package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-rails",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
since it drops compatibility with a browser I think this should be a major version upgrade

Reply:
Updated